src/index.js

```diff
@@ -4,6 +4,7 @@ import './index.css';
 import Layout from "./components/Layout";
 import * as serviceWorker from './serviceWorker';
 import { BrowserRouter } from "react-router-dom";
+import styled from "styled-components";
```

Comment:
Please try to keep your code clean - remove redundant code.
![image](https://user-images.githubusercontent.com/52723192/104512404-31798100-55ff-11eb-9dfb-2b81cc7b3cd8.png)
Also, if there are any other console warnings - ideally they should not be ignored but lead to actions.